test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -42,4 +44,71 @@ def test_dlc_major_version_label(image, region):
 
     assert major_version, f"{image} has no LABEL named 'dlc_major_version'. Please insert label."
 
-    LOGGER.info(f"{image} has 'dlc_major_version' = {major_version}")
+    test_utils.LOGGER.info(f"{image} has 'dlc_major_version' = {major_version}")
+
+
+@pytest.mark.integration("dlc_major_version_label")
+@pytest.mark.model("N/A")
+def test_dlc_version_dockerfiles(image):
+    """
+    Test to make sure semantic versioning scheme in Dockerfiles is correct
+
+    :param image: <str> ECR image URI
+    """
+    dlc_dir = os.getcwd().split(f"{os.sep}test{os.sep}")[0]
+    job_type = test_utils.get_job_type_from_image(image)
+    framework, fw_version = test_utils.get_framework_and_version_from_tag(image)
+    processor = test_utils.get_processor_from_image_uri(image)
+
+    root_dir = os.path.join(dlc_dir, framework, job_type, "docker")
+
+    # Skip older FW versions that did not use this versioning scheme
+    references = {"tensorflow2": "2.2.0", "tensorflow1": "1.16.0", "mxnet": "1.7.0", "pytorch": "1.5.0"}
+    if test_utils.is_tf1(image):
+        reference_fw = "tensorflow1"
+    elif test_utils.is_tf2(image):
+        reference_fw = "tensorflow2"
+    else:
+        reference_fw = framework
+    if processor != "eia" and fw_version < references[reference_fw]:
+        pytest.skip(
+            f"Not enforcing new versioning scheme on old image {image}. "
+            f"Started enforcing version scheme on the following: {references}"
+        )
```

Comment:
Use [packaging.package.Version](https://packaging.pypa.io/en/latest/version/#packaging.version.Version) to compare version numbers using `<`, `>`, `=`, and `!=`.

This code is not future proof, and can fail when you compare `1.10` to `1.9`, for example.

For example:

https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/sanity/test_safety_check.py#L118-L124

Reply:
Thanks for the suggestion, will check this out